cli/cmd/cluster.go

```diff
@@ -135,8 +247,46 @@ func (cmd *PxfCommand) GenerateOutput(clusterData *ClusterData) error {
 		}
 		response += fmt.Sprintf("\n%s ==> %s", host, errorMessage)
 	}
+	return response
+}
 
-	return fmt.Errorf(response)
+func logCommandOutput(output *cluster.RemoteOutput) {
+	if isDebugEnabled() {
+		gplog.Debug("Remote command output - scope=%d", output.Scope)
+		for _, command := range output.Commands {
+			hasFailed := command.Error != nil
+			gplog.Debug("host=%s, command=%q, failed=%t, stdout=%q", command.Host, command.CommandString, hasFailed, command.Stdout)
+			if hasFailed {
+				gplog.Debug("    stderr=%q", command.Stderr)
```

Comment:
why do we have so many spaces in front? 
```suggestion
				gplog.Debug(" stderr=%q", command.Stderr)
```

Reply:
this is intentional, since this is a continuation of the previous message, but it goes on a separate line, it is indented with spaces.